x-pack/plugins/security/server/authentication/api_key.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface CreateApiKeyOptions {
+  callCluster: any;
```

Comment:
__nit:__ `callCluster: any;` ---> `callAsCurrentUser: ScopedClusterClient['callAsCurrentUser'];` or just pass entire `ClusterClient` and make `request: KIbanaRequest` a part of `createAPIKey` signature. Up to you.